NetworkIOMeter.c

```diff
@@ -28,8 +28,10 @@ static const int NetworkIOMeter_attributes[] = {
 
 static MeterRateStatus status = RATESTATUS_INIT;
 static uint32_t cached_rxb_diff;
+static char cached_rxb_diff_str[6];
 static uint32_t cached_rxp_diff;
 static uint32_t cached_txb_diff;
+static char cached_txb_diff_str[6];
```

Comment:
Alignment on these two cached strings is 8 bytes anyway, thus why not use them completely?

Reply:
I thought 6 bytes are sufficient for Meter_humanUnit() output, since the function would never print more than 5 characters (or 6 characters if we include `\0`).